src/main/java/org/broadinstitute/consent/http/resources/DarCollectionResource.java

```diff
@@ -202,22 +207,17 @@ public Response cancelDarCollectionByCollectionId(
         }
       }
 
-      DarCollection cancelledCollection;
-      switch (actingRole) {
-        case ADMIN:
-          cancelledCollection = darCollectionService.cancelDarCollectionElectionsAsAdmin(
-              collection);
-          break;
-        case CHAIRPERSON:
-          cancelledCollection = darCollectionService.cancelDarCollectionElectionsAsChair(collection,
-              user);
-          break;
-        default:
+      DarCollection cancelledCollection = switch (actingRole) {
+        case ADMIN -> darCollectionService.cancelDarCollectionElectionsAsAdmin(collection);
+        case CHAIRPERSON -> darCollectionService.cancelDarCollectionElectionsAsChair(collection, user);
+        default -> {
           validateUserIsCreator(user, collection);
-          cancelledCollection = darCollectionService.cancelDarCollectionAsResearcher(collection);
-          break;
-      }
-
+          yield darCollectionService.cancelDarCollectionAsResearcher(collection);
+        }
+      };
+      ComplianceLogger.getInstance()
+          .logDARCancellation(user, cancelledCollection.getDatasets().stream().toList(),
```

Comment:
If every use of `ComplianceLogger` requires `.getInstance()` I would make `getInstance()` private and provide static methods which call `getInstance()` internally.